charts/haproxy/values.yaml

```diff
@@ -177,3 +184,14 @@ affinity: {}
 livenessProbe: {}
 
 readinessProbe: {}
+
+## Additional annotations
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+annotations: {}
+#  key: value
+podAnnotations: {}
+#  key: value
+
+## Extra volumes
+extraVolumes: []
```

Comment:
please add here some example in comments, with the expected structure.

Reply:
> please add here some example in comments, with the expected structure.

done!